internal/alb/rs/rule.go

```diff
@@ -36,6 +36,8 @@ type NewDesiredRuleOptions struct {
 // NewDesiredRule returns an rule.Rule based on the provided parameters.
 func NewDesiredRule(o *NewDesiredRuleOptions) (*Rule, error) {
 	r := &elbv2.Rule{
+		IsDefault: aws.Bool(false),
```

Comment:
What's is the `IsDefault`'s original purpose? Since it will never be true again..and there is still code referencing it.